decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+  * `slot:id` will be the identifier of the `instrument_used` and `vendor` and `model` will NOT be inlined for `Class:DataGeneration`. Rather, it will be inferred / traced using the instrument's id.
+* Schema support for complex paths of data generation and data processing.
+  * `Class:DataGeneration` can have instances where a single sample has multiple data files that need processed together during WorkflowExecution.  
+  * Relationships between samples and the data objects will be captured using `slot:part_of`, linking the 'child' back to the 'parent'.
+
+### Class:WorkflowExecution & Class:WorkflowChain
+* `Class:WorkflowExecution` will be a subclass of `Class:PlannedProcess`
+* `Class:WorkflowExecution` will replace `Class:WorkflowExecutionActivity` and will be a subclass of `Class:PlannedProcess`
+* To group subclasses of `Class:WorkflowExecution` into implemented steps, `Class:WorkflowChain` was created.
+  * This also provides clear connection back to the DataGeneration output, using `was_informed_by`
+    * `was_informed_by` is being removed from `Class:WorkflowExecution`
+  * Any implementation of `Class:WorkflowExecution` will use `Class:WorkflowChain`. Even when it's a single linked chain (ie implementation of a single subclass of `Class:WorkflowExecution`)
+* `Class:WorkflowExecution` will provide `part_of` with `Range:Class:WorkflowChain`
+  * This replaces 'used' and will no longer be on `Class:WorkflowExecution`
+* Output from `WorkflowExecution` will always have a value on `slot:data_category` of `processed_data` of `Class:DataObject`
+* `slot:analyte_category` will be used to inform the type of data that is being generated and can be instantiated on `Class:DataGeneration` and `Class:WorkflowChain`
+  * `slot:omics_type` will be removed and replaced with `slot:analyte_category` to denote the type of data generated.
+* `slot:gold_analysis_project_identifiers` and `slot:jgi_analysis_project_identifiers` will be included on `Class:WorkflowChain`
+
+### Other
+* `Class:DataObject` will NOT have subclasses. Rather, it will have `slot:data_category` which has a range `DataCategoryEnum`
+* `slot:data_object_type` will be renamed to `slot:data_object_category`. 
+* Slots associated with `Class:Biosample` or `Class:OmicsProcessing` that were incorrectly associated, were removed and added to more appropriate classes. (ex: `slot:pcr_primers` was removed from `Class:Biosample` and added to the new `Class:LibraryPrep`)
+* `Class:Reaction` will renamed to `Class:BiochemicalReaction` for clarity between this class and `Class:ChemicalConversionProcess`.
+* Subclasses will have their own typecodes.
+  * This may mean that eventually, if classes are renamed, typecodes might not make sense.
+* Reciprocal or redundant relationship slots (`has_part` + `part_of`, `was_informed_by` + `used`) will not be instantiated. Only a single direction should be provided.
+* All classes should have `slot:type` with `Range:CURIE` and `designates_type = True`
```

Comment:
Most of the other bullet points use the word "will" or "is", while this one and the one below it use "should".